lib/network.sh

```diff
@@ -118,6 +110,23 @@ else
     exit 1
 fi
 
+if [[ -n "${CLUSTER_APIENDPOINT_IP}" ]]; then
+  # Accept user provided value if set
+  export CLUSTER_APIENDPOINT_IP
+elif [[ -n "${EXTERNAL_SUBNET_V4}" ]]; then
+  network_address CLUSTER_APIENDPOINT_IP "${EXTERNAL_SUBNET_V4}" 249
+else
+  network_address CLUSTER_APIENDPOINT_IP "${EXTERNAL_SUBNET_V6}" 249
```

Comment:
I was not aware of this ipv6 IP being used for v6 family.., though we never test this in the CI and it looks to be not allocated in other places: https://github.com/metal3-io/metal3-dev-env/blob/681b4d26ab5f90b0d08679df36b82ebb74eca993/lib/network.sh#L152 , https://github.com/metal3-io/metal3-dev-env/blob/681b4d26ab5f90b0d08679df36b82ebb74eca993/lib/network.sh#L122

Reply:
It was more of a like question / observation point I had, and not a blocker. I was also assuring that 249 IP is not used anywhere so it is safe to assign it.